css/styles.css

```diff
@@ -0,0 +1,136 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+body {
+  font-family: Verdana, sans-serif;
+  background-color: #f0f0f0;
+}
+
+nav {
+  width: 100%;
+}
+
+.container {
+  width: 100%;
+  margin: 0 auto;
+  padding: 0;
+}
+
+.head-background {
+  background-color: #562a81;
+  width: 100%;
+  height: 5rem;
+}
+
+#code-journal-header {
+  color: white;
+  font-family: "Gill Sans", sans-serif;
+  font-size: 22px;
+  font-weight: lighter;
+  letter-spacing: 0.1rem;
+  padding: 1.75rem 1rem;
+}
+
+#left-column {
+  padding-bottom: 1.5rem;
+}
+
+#new-entry {
+  margin: 0 1rem;
+}
+
+#entry-background {
+  min-height: 100vh;
+  max-width: 900px;
```

Comment:
Works, but we already have a `max-width` set on the `.container` class, so we can just utilize that class on an element to contain the `width`